Jellyfin.Api/Controllers/SystemController.cs

```diff
@@ -71,6 +71,19 @@ public SystemController(
     public ActionResult<SystemInfo> GetSystemInfo()
         => _systemManager.GetSystemInfo(Request);
 
+    /// <summary>
+    /// Gets information about the server.
+    /// </summary>
+    /// <response code="200">Information retrieved.</response>
+    /// <response code="403">User does not have permission to retrieve information.</response>
+    /// <returns>A <see cref="SystemInfo"/> with info about the system.</returns>
+    [HttpGet("StorageResources")]
+    [Authorize(Policy = Policies.FirstTimeSetupOrElevated)]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    public ActionResult<SystemStorageInfo> GetSystemStorageInfo()
```

Comment:
```suggestion
    public ActionResult<SystemStorageInfo> GetSystemResources()
```
(if we rename the path)